deepchem/models/brics_generator.py

```diff
@@ -0,0 +1,214 @@
+from rdkit.Chem.BRICS import BRICSDecompose, BRICSBuild
+from typing import Tuple
+from rdkit import Chem
+
+
+class BRICSGenerator:
```

Comment:
This class feels like it should go into utils. We can set up a new deepchem.utils.polymers possibly